docs/STUMP_Tiles_Algorithm_for_Cache_Optimization.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Instead of <code>arr[i]</code> , maybe we actually set the elements to a specific values like <code>arr[i] = i</code> . This way, some work is actually being done and the elapsed time is more pronounced</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>